src/main/java/core/basesyntax/model/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.model;
+
+import core.basesyntax.model.abstraction.Color;
+
+public class Rectangle extends Square {
```

Comment:
```suggestion
public class Rectangle extends Square {
```
Rectangle **IS NOT** a Square
Square **IS** a Rectangle (with equals sides)
Making `class Square extends Rectangle ` is possible, but not a good idea.